cmake/std/atdm/README.md

```diff
@@ -1067,10 +1068,36 @@ CTest runs everything using the `jsrun` command. You must run jsrun from a
 compute node which can be acquired using the above bsub command.
 
 Once you're on a compute node, you can run ctest. For example:
+
 ```bash
 $ ctest -j4
 ```
 
+The MPI test exectuables are run by a wrapper script `trilinos_jsrun` which
```

Comment:
I would also like to know how one would check which value of TPETRA_ASSUME_CUDA_AWARE_MPI was used in a particular test configuration.  If one wants to reproduce a failing test, where should one look in CDash to get the value used for that test?  The environment variable setting is not archived in the CMake configuration output.

Reply:
> I would also like to know how one would check which value of TPETRA_ASSUME_CUDA_AWARE_MPI was used in a particular test configuration.

It is printed out by `trilinos_jsrun` before it runs `jsrun`.  Therefore, that information is on CDash in the detailed test output.  For example, if you  if you look at the output for the test ` TpetraCore_Behavior_Default_MPI_4` [here](https://testing-dev.sandia.gov/cdash/testDetails.php?test=85015522&build=5264563) you will see:

```
BEFORE: jsrun  '-p' '4' '--rs_per_socket' '4' '/vscratch1/jenkins/vortex-slave/workspace/Trilinos-atdm-ats2-cuda-10.1.243-gnu-7.3.1-spmpi-2019.06.24_static_dbg/SRC_AND_BUILD/BUILD/packages/tpetra/core/test/Behavior/TpetraCore_Behavior_Default.exe'
AFTER: export TPETRA_ASSUME_CUDA_AWARE_MPI=0; jsrun  '-p' '4' '--rs_per_socket' '4' '/vscratch1/jenkins/vortex-slave/workspace/Trilinos-atdm-ats2-cuda-10.1.243-gnu-7.3.1-spmpi-2019.06.24_static_dbg/SRC_AND_BUILD/BUILD/packages/tpetra/core/test/Behavior/TpetraCore_Behavior_Default.exe'
```

If you compare that to the CUDA-aware running of that same test [here](https://testing-dev.sandia.gov/cdash/testDetails.php?test=85029275&build=5265702) you see:

```
BEFORE: jsrun  '-p' '4' '--rs_per_socket' '4' '/vscratch1/jenkins/vortex-slave/workspace/Trilinos-atdm-ats2-cuda-10.1.243-gnu-7.3.1-spmpi-2019.06.24_static_dbg/SRC_AND_BUILD/BUILD/packages/tpetra/core/test/Behavior/TpetraCore_Behavior_Default.exe'
AFTER: export TPETRA_ASSUME_CUDA_AWARE_MPI=1; jsrun  '-E LD_PRELOAD=/usr/tce/packages/spectrum-mpi/ibm/spectrum-mpi-2019.06.24/lib/pami_451/libpami.so' '-M -gpu' '-p' '4' '--rs_per_socket' '4' '/vscratch1/jenkins/vortex-slave/workspace/Trilinos-atdm-ats2-cuda-10.1.243-gnu-7.3.1-spmpi-2019.06.24_static_dbg/SRC_AND_BUILD/BUILD/packages/tpetra/core/test/Behavior/TpetraCore_Behavior_Default.exe'
```

Hopefully that is clear.

Reply:
Right; that information should be in the documentation.  Trilinos developers are not accustomed to looking for that information, and it is specialized to the ATS-2 builds.